DateTimeCalendar/Time.java

```diff
@@ -0,0 +1,90 @@
+public class Time {
+    private static final int SEC_IN_MIN = 60;
+    private static final int SEC_IN_HR = 3600;
+
+    public static long convToSeconds(int hours, int minutes, int seconds){
+        return ((hours*SEC_IN_HR) + (minutes*SEC_IN_MIN)) + seconds;
+    }
+
+    //TODO:
+    //[] Add method for Equals, must override from obj
+
+    private final long seconds;
+
+    private Time(){
+        this.seconds = 0;
+    }
+
+    public Time(long seconds){
+        this.seconds = seconds;
+    }
+
+    public Time(int hour, int minute, int second){
+        this.seconds = convToSeconds(hour, minute, second);      
+    }
+
+    public Time add(Time input){
+        return new Time(this.seconds + input.seconds);
+    }
+
+    public Time add(int hours, int minutes, int seconds){
+        return new Time(this.seconds + convToSeconds(hours, minutes, seconds));
+    }
+
+    public Time subtract(Time input){
+        return new Time(this.seconds - input.seconds);
+    }
+
+    public Time subtract(int hours, int minutes, int seconds){
+        return new Time(this.seconds - convToSeconds(hours, minutes, seconds));
+    }
+
+    public int compareTo(Time input){
+        if(this.seconds == input.seconds) {
+            return 0;
+        } else if(this.seconds > input.seconds) {
+            return 1;
+        } else {
+            return -1;
+        } 
```

Comment:
Java trick. The Long class provides a static method to compare 2 long values: https://docs.oracle.com/javase/7/docs/api/java/lang/Long.html#compare(long,%20long)
```java
return Long.compare(this.seconds, input.seconds);
```

Reply:
That's much easier to read, thank you!